app/profile/page.tsx

```diff
@@ -0,0 +1,303 @@
+            size="sm"
+            colorScheme="quaternary"
+          >
+            Joined {userFirstParticipationDate}
+          </Tag>
+        )}
+      </div>
+      <div className="grid w-full grid-cols-1 justify-between gap-4 sm:grid-cols-2 lg:grid-cols-4">
+        <StatsCard
+          title="Positions value"
+          value={positionValue}
+          symbol="usd"
+          isLoading={isLoadingUserPositions}
+        />
+        <StatsCard
+          title="Volume traded"
+          value={volumeTraded}
+          symbol="usd"
+          isLoading={isLoadingUserPositions}
+        />
+        <StatsCard
+          title="Success rate"
+          value={succesRate}
+          symbol="%"
+          isLoading={isLoadingUserPositions}
+        />
+        <StatsCard
+          title="Profit/loss"
+          value={profitLoss}
+          symbol="usd"
+          isLoading={isLoadingUserPositions}
+        />
+      </div>
+      <TabGroup>
+        <TabHeader className="w-full overflow-x-auto md:overflow-x-visible">
+          <BetsListTab bets={userPositions ?? []}>All Bets</BetsListTab>
+          <BetsListTab bets={filterActiveBets}>Active</BetsListTab>
+          <BetsListTab bets={filterCompleteBets}>Complete</BetsListTab>
+        </TabHeader>
+        <TabBody className="mt-8 w-full">
+          <ProfileBetsListPanel bets={userPositions ?? []} isLoading={isLoading} />
+          <ProfileBetsListPanel
+            emptyText="No active bets"
+            bets={filterActiveBets}
+            isLoading={isLoading}
+          />
+          <ProfileBetsListPanel
+            emptyText="No complete bets"
+            bets={filterCompleteBets}
+            isLoading={isLoading}
+          />
+        </TabBody>
+      </TabGroup>
+    </main>
+  );
+};
+
+const ProfileBetsListPanel = (props: BetsListPanelProps) => (
+  <BetsListPanel {...props} CardComponent={ProfileCardBet} />
+);
+
+const StatsCard = ({
+  title,
+  value,
+  symbol,
+  isLoading,
+}: {
+  title: string;
+  value: string;
+  symbol: string;
+  isLoading?: boolean;
+}) => {
+  if (isLoading) return <StatsCardLoading />;
+
+  return (
+    <div className="w-full space-y-2 rounded-16 bg-surface-surface-0 p-6 font-semibold text-text-low-em ring-1 ring-outline-base-em">
+      <div className="text-xs font-bold uppercase">{title}</div>
+      <div className="text-2xl">
+        <span className="text-text-high-em">{value}</span>{' '}
+        <span className="uppercase">{symbol}</span>
+      </div>
+    </div>
+  );
+};
+
+const StatsCardLoading = () => (
+  <div className="w-full space-y-2 rounded-16 bg-surface-surface-0 p-6 font-semibold text-text-low-em ring-1 ring-outline-base-em">
+    <div className="h-3 w-20 animate-pulse rounded-8 bg-outline-low-em" />
+    <div className="flex space-x-2">
+      <div className="h-9 w-28 animate-pulse rounded-8 bg-outline-low-em" />
+      <div className="h-9 w-10 animate-pulse rounded-8 bg-outline-low-em" />
+    </div>
```

Comment:
I would remove this one

Reply:
I like this spacing + div since it represents the symbol and looks more like the StatsCard component. One thing we can do is to just "load" the number part and show the title and symbol content. What do you think?

Reply:
not sure, would have to try it to have opinion, but go ahead and see if that feels better.